.github/workflows/cxx.yml

```diff
@@ -1,83 +1,97 @@
-# Copyright © 2024 Bret Brown
-# SPDX-License-Identifier: MIT
-
-name: Test
+name: Test Multi-Platform
 
 on:
   push:
     branches: [ main ]
-    paths:
```

Comment:
Please remove:
https://github.com/beman-project/exemplar/blob/d9145b52c9072a569d747547edfc7e03a1494809/.github/workflows/ci_tests.yml#L9-L10

If you intend to merge this change.